packages/react/.storybook/Container.js

```diff
@@ -24,11 +24,8 @@ export default class Container extends Component {
         <div
           data-floating-menu-container=""
           role="main"
+          className="storybook-container"
```

Comment:
Noticed you put the class name as `.storybook-center-container` in the corresponding scss file?

Reply:
Great catch, this was leftover from trying to figure out a way to flex center smaller components while allowing full width for something like this footer. I'll remove this, though. thanks.